.github/ISSUE_TEMPLATE.md

```diff
@@ -1,40 +1,47 @@
 <!--- Provide a general summary of the issue in the Title above -->
 
-<!--- DO NOT PASTE ONLY IMAGES OF TERMINAL OUTPUT AS AN ISSUE --->
+<!--
+*************************************************************************************
+*************************************************************************************
+  Make sure you are running the latest version of Ganache before you open an issue.  
```

Comment:
Maybe we can link to the current releases for ultimate ease https://github.com/trufflesuite/ganache/releases